modules/custom/drupalmel_core/drupalmel_core.install

```diff
@@ -76,30 +71,24 @@ function drupalmel_core_update_7000() {
 
   // Run this modules persistent update hook.
   drupalmel_core_persistent_update();
+}
 
-  // Add blog feeds.
+/**
+ * Adds blog feeds.
+ */
+function drupalmel_core_update_7001() {
```

Comment:
Is there any reason you are changing a hook_update_N() after it has been run? While your change might be more succinct and optimized, it is not how the history of this project went. At one point, the current 7000 was run, and at a later point the current 7001 was run. At no point where either of these new version of the hook_update_N() functions actually run.

Furthermore, while some of the change may be related to coding standards, the re-ordering of the functions is not.

Can you remove this change please?